frontend/src/api/k8s/roleBindings.ts

```diff
@@ -63,6 +74,53 @@ export const generateRoleBindingServingRuntime = (namespace: string): RoleBindin
   return roleBindingObject;
 };
 
+export const generateRoleBindingProjectSharing = (
```

Comment:
The two options here would be have two roledindings and multiple subjects, or create rolebidnigns for each permission, I opted for the second to have the creation date.